test/CMakeLists.txt

```diff
@@ -149,3 +149,9 @@ ADD_TEST( t_SensThickness_Clic_o2_v4 "${CMAKE_INSTALL_PREFIX}/bin/run_test_${Pac
           ${CMAKE_INSTALL_PREFIX}/bin/TestSensThickness ${CMAKE_CURRENT_SOURCE_DIR}/../CLIC/compact/CLIC_o2_v04/CLIC_o2_v04.xml 300 50 )
 ADD_TEST( t_SensThickness_CLIC_o3_v15 "${CMAKE_INSTALL_PREFIX}/bin/run_test_${PackageName}.sh"
           ${CMAKE_INSTALL_PREFIX}/bin/TestSensThickness ${CMAKE_CURRENT_SOURCE_DIR}/../CLIC/compact/CLIC_o3_v15/CLIC_o3_v15.xml 100 50 )
+
+#--------------------------------------------------
+# check if files named the same contain the same in FCCee
+SET( test_name "test_files_versions_FCCee" )
+ADD_TEST( t_${test_name} "${CMAKE_INSTALL_PREFIX}/bin/run_test_${PackageName}.sh" "${CMAKE_SOURCE_DIR}/utils/compareIdenticalFiles.sh ${CMAKE_SOURCE_DIR}/FCCee ${CMAKE_SOURCE_DIR}/utils/IdenticalFiles_ToBeIgnored.txt" )
```

Comment:
`CMAKE_SOURCE_DIR` depends on where this project is built from, use `PROJECT_SOURCE_DIR` or `CMAKE_CURRENT_SOURCE_DIR` like in the other tests.

Reply:
done!